css/store.css

```diff
@@ -0,0 +1,60 @@
+
+.main{
+    display: flex;
+    margin-left: 150px;
+    gap:100px;
+}
+img{
```

Comment:
Please use class names next time, instead of tags

I would also apply the width and height to the container `.img_container` instead of the image itself. It is even better if the width is 50% on the container. The image itsef should have a width of 100%, which follows the container's width